recipes/abinit/meta.yaml

```
@@ -0,0 +1,70 @@
+{% set name = "abinit" %}
+{% set version = "8.10.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/abinit/abinit/archive/{{ version }}.tar.gz
+  sha256: b362bda0ccefbc7254d653d551ba1952aa4a92f9f845208caa3446cb8960ad76
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - m4
+    - autoconf
+    - automake
+    - openmpi-mpifort  # [mpi == 'openmpi']
+    - {{ compiler('fortran') }}
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - hdf5 * nompi_*
+    - libnetcdf * nompi_*
+    - netcdf-fortran
+    - libxc
+    - {{ mpi }}
+    - libblas
+    - libcblas
+    - liblapack
+    - liblapacke
+    - scalapack
+  run:
+    - hdf5 * nompi_*
```

Won't this remove the version constraint from `run_exports`?
```suggestion
    - hdf5 * nompi_*
```
If these is documented somewhere point me to it else if you don't understand the issue I'm referring to ask and I'll write a better comment explaining.